[esx]/esx_context/main.lua

```diff
@@ -1,19 +1,47 @@
 local defaultPosition = "right" -- [ left | center | right ]
-local activeMenu
+local activeMenu, keepControl
 local Debug = ESX.GetConfig().EnableDebug
 
 -- Global functions
 -- [ Post | Open | Closed ]
 
+local function NuiFocus(state, keepInput)
+	SetNuiFocus(state, state)
+	if keepInput then
+		SetNuiFocusKeepInput(state)
+		CreateThread(function()
+			while keepControl do
+				DisableControlAction(0, 1, true)
+				DisableControlAction(0, 2, true)
+				DisableControlAction(0, 4, true)
+				DisableControlAction(0, 6, true)
+				DisableControlAction(0, 12, true)
+				DisableControlAction(0, 13, true)
+				DisableControlAction(0, 24, true)
+				DisableControlAction(0, 25, true)
+				DisableControlAction(0, 66, true)
+				DisableControlAction(0, 67, true)
+				DisableControlAction(0, 69, true)
+				DisableControlAction(0, 70, true)
+				DisableControlAction(0, 91, true)
+				DisableControlAction(0, 92, true)
+				DisableControlAction(0, 95, true)
+				DisableControlAction(0, 98, true)
+				Wait(0)
+			end
+		end)
+	end

```

Comment:
````suggestion
	SetNuiFocusKeepInput(keepInput)
	if not keepInput then
		return
	end
	CreateThread(function()
		while keepControl do
			DisableControlAction(0, 1, true)
			DisableControlAction(0, 2, true)
			DisableControlAction(0, 4, true)
			DisableControlAction(0, 6, true)
			DisableControlAction(0, 12, true)
			DisableControlAction(0, 13, true)
			DisableControlAction(0, 24, true)
			DisableControlAction(0, 25, true)
			DisableControlAction(0, 66, true)
			DisableControlAction(0, 67, true)
			DisableControlAction(0, 69, true)
			DisableControlAction(0, 70, true)
			DisableControlAction(0, 91, true)
			DisableControlAction(0, 92, true)
			DisableControlAction(0, 95, true)
			DisableControlAction(0, 98, true)
			Wait(0)
		end
	end)

Reply:
SetNuiFocusKeepInput(keepInput) should be under the return too

Reply:
I thought about this because that's how the focus keep input is restored, but maybe it's not like that